project/plugins.sbt

```diff
@@ -10,7 +10,7 @@ addSbtPlugin("org.scalastyle"   %% "scalastyle-sbt-plugin" % "0.8.0")
 addSbtPlugin("com.typesafe.sbt"  % "sbt-native-packager"   % "1.1.4")
 addSbtPlugin("com.typesafe.sbt"  % "sbt-git"               % "0.8.5")
 addSbtPlugin("de.heikoseeberger" % "sbt-header"            % "2.0.0")
-addSbtPlugin("org.wartremover"   % "sbt-wartremover"       % "2.1.1")
+addSbtPlugin("org.wartremover"   % "sbt-wartremover"       % "2.1.1") // 2.2.0 PublicInference fails
```

Comment:
There is a bug in WartRemover 2.2.0 which crashes the compiler when checking for inferred types on public members. Because this check is important we should stick with 2.1.1 for now.